packages/perseus-editor/src/widgets/free-response-editor.tsx

```diff
@@ -32,18 +39,145 @@ class FreeResponseEditor extends React.Component<Props> {
+            scoringCriteria: this.props.scoringCriteria.filter(
+                (_, i) => i !== index,
+            ),
+        });
+    };
+
+    handleAddCriterion = () => {
+        this.props.onChange({
+            scoringCriteria: [...this.props.scoringCriteria, {text: ""}],
+        });
+    };
+
+    renderCriteriaList = () => {
+        return this.props.scoringCriteria.map((criterion, index) => {
+            return (
+                <CriterionEditor
+                    criterion={criterion}
+                    index={index}
+                    key={index}
+                    numCriteria={this.props.scoringCriteria.length}
+                    onChange={this.handleUpdateCriterion}
+                    onDelete={this.handleDeleteCriterion}
+                />
+            );
+        });
+    };
+
     render(): React.ReactNode {
         return (
-            <LabeledTextField
-                label={"Question"}
-                value={this.props.question}
-                onChange={(question: string) => this.props.onChange({question})}
-                style={{
-                    marginBottom: spacing.large_24,
-                }}
-            />
+            <View>
+                <label className={css(styles.questionContainer)}>
+                    <HeadingSmall>Question</HeadingSmall>
+                    <textarea
+                        value={this.props.question}
+                        onChange={(e) =>
+                            this.props.onChange({question: e.target.value})
+                        }
+                    />
+                </label>
+                <View>
+                    <HeadingSmall>Scoring criteria</HeadingSmall>
+                    <View style={styles.criteriaList}>
+                        {this.renderCriteriaList()}
+                    </View>
+                    <View>
+                        <Button
+                            onClick={this.handleAddCriterion}
+                            startIcon={plusCircleIcon}
+                        >
+                            Add an item
+                        </Button>
+                    </View>
+                </View>
+            </View>
         );
     }
 }
 
+const CriterionEditor = function (props: {
+    index: number;
+    numCriteria: number;
+    criterion: PerseusFreeResponseWidgetScoringCriterion;
+    onChange: (
+        index: number,
+        criterion: PerseusFreeResponseWidgetScoringCriterion,
+    ) => void;
+    onDelete: (index: number) => void;
+}): React.ReactNode {
+    const isOnlyCriterion = props.numCriteria <= 1;
+
+    return (
+        <View style={styles.criterionContainer}>
+            <textarea
+                data-testid={`criterion-input-${props.index}`}
+                onChange={(e) =>
+                    props.onChange(props.index, {
+                        text: e.target.value,
+                    })
+                }
+                value={props.criterion.text}
+            />
```

Comment:
To be accessible, this textarea should have a unique label, like "Text of Criterion 1". If it had such a label, we wouldn't need the test ID.